packages/webpack-scaffold/README.md

```diff
@@ -86,7 +86,7 @@ this.configuration.myScaffold.webpackOptions.entry = createDynamicPromise(["app.
 
 Param: `String`
 
-Used to create an [assetFilterFunction](https://webpack.js.org/configuration/performance/#performance-assetfilter)
+Used to create a [assetFilterFunction](https://webpack.js.org/configuration/performance/#performance-assetfilter)
```

Comment:
The former was fine.

Reply:
See its readme context

> The former was fine.

![Screenshot 2020-02-22 13:28:31](https://user-images.githubusercontent.com/35539313/75088704-0dd8c280-5577-11ea-8c92-b1984fd50dc9.png)

Reply:
@mohitm15 you should be making the change [here](https://github.com/webpack/webpack-cli/tree/next/packages/webpack-scaffold#createexternalfunction)

Reply:
Instead you made the change here:
![Screenshot from 2020-02-22 14-54-43](https://user-images.githubusercontent.com/25279263/75089910-7f1e7280-5583-11ea-87c6-405be1c0278c.png)

Reply:
Thanks for correcting @jamesgeorge007